web_widget_kanban_many2one/static/src/js/web_widget_kanban_many2one.js

```diff
@@ -0,0 +1,62 @@
+odoo.define('web.kanban_editable', function(require){
+
+    var AbstractField = require('web.AbstractField');
+    var core = require('web.core');
+    var field_registry = require('web.field_registry');
+
+    var _t = core._t;
+
+    var many2one_kanban = AbstractField.extend({
+        template: _t('Kanban_editable_temp'),
+        events: {
+            'click .selectM2MField': '_onSelectField',
+        },
+
+        init: function(){
+            this._super.apply(this, arguments);
+            if(!this.recordData[this.name])
+                this._updateField(this.recordData[this.name].id);
+            else
+                this._updateField(this.recordData[this.name].data.id);
+        },
+        _onSelectField: function(){
+            var self = this;
+            var val = this.$el.find('select[name="id"]').val();
+            var field_name = this.name;
+            var params = [[this.res_id],{
+                [field_name]: parseInt(val)
+            }];
+            this._rpc({
+                model: this.model,
+                method: 'write',
+                args: params
+            }).then(function(result){});
+            },
```

Comment:
This `.then()` is superfluous, it will just call a dummy function, so the whole `.then(...)` can be removed